### PR TITLE
Empty scene setup

### PR DESCRIPTION
**Pull request by aaronabraham311**
- Added buttons for empty scene as described in Notion
- Design is minimal (and might I add not great) but functionality is present
- Blocked by #29 

Screen Shot 2021-04-18 at 2 22 21 PM

### CODE REVIEW

File: src/Components/Admin/EnvironmentEditor/environment.js [hunk 1]
```diff
@@ -19,6 +47,26 @@ export default function EnvironmentEditor() {
...
+                    <div className={classes.buttonContainer}>
+                        <Button
+                            startIcon={<AccountBalanceIcon />}
+                            className={classes.button}
+                        >
+                            New Scene from Template
+                        </Button>
+                    </div>
+                </div>
+            </div>
```
Comment (ahmedhamodi) [inline src/Components/Admin/EnvironmentEditor/environment.js:69]: Can we load this dynamically if a prop (arbitrarily called data) is null / empty? Not exactly sure what the structure of the data prop will be, but null should be a safe thing to check against.

Comment (ahmedhamodi) [reply src/Components/Admin/EnvironmentEditor/environment.js:69]: Thinking a bit more, `scenes` would be a better name and then on the call that goes to this page, we can pass in scenario.scenes, which will default to an empty array if no scenes. so the conditional rendering should be based on `scenes !== []`.

Comment (aaronabraham311) [reply src/Components/Admin/EnvironmentEditor/environment.js:69]: Was planning on doing this for the next ticket, unless you would like it implemented now.

Comment (ahmedhamodi) [reply src/Components/Admin/EnvironmentEditor/environment.js:69]: Sure next ticket is fine. As long as you are aware

**Review by ahmedhamodi — APPROVED** [review]

**Pull request merged**